ui/chat/src/discussion.ts

```diff
@@ -89,11 +89,8 @@ function renderInput(ctrl: ChatCtrl): VNode | undefined {
 }
 
 function prependChatInput(chatInput: HTMLInputElement, prefix: string): void {
-  if (chatInput.value.startsWith(prefix)) return;
-  const newVal = prefix + chatInput.value;
+  if (!chatInput.value.startsWith(prefix)) chatInput.value = prefix + chatInput.value;
   chatInput.focus();
-  chatInput.select();
```

Comment:
not sure what that was for? didn't seem useful to me to have the entire input text selected. 

I might be missing something